src/config/routes.js

```diff
@@ -16,6 +16,7 @@ import Donate from 'pages/Donate/Donate';
 import Geneology from 'pages/Geneology/Geneology';
 import GetInvolved from 'pages/GetInvolved/GetInvolved';
 import Demo from 'pages/Demo/Demo';
+import AboutMissionAndPurpose from 'pages/About/MissionAndPurpose/MissionAndPurpose';
```

Comment:
Let's change the directory structure to make `missionAndPurpose` a child of `pages` instead of `About`